unthink-stack/src/server/framework/service-demo.ts

```diff
@@ -61,23 +62,25 @@ const sd: ServiceDefinition = {
 
       return result;
     },
-/*
-    // This would throw an error because of duplicate 'GET'
-    'GET': (ctx: HandlerContext): Promise<ServiceResult> => {
-    ...
-
-    }
-*/
-/*
-    // This will throw a compiler error because 'BAD' is not valid option for the key
-    'BAD': (ctx: HandlerContext): Promise<ServiceResult> => {
-    ...
-
-    }
-*/
   }
 };
 
+/****** For templates ******/
+
+const td: TemplateDefintion = {
+  '/home': {
+    // Virtually no need for a class here just can directly map request to a view
+    'GET': async (ctx: HandlerContext): Promise<TemplateResult> => {
+      return TemplateResult.view('home');
+    },
+  },
+  '/login': {
+    // Virtually no need for a class here just can directly map request to a view
+    'GET': async (ctx: HandlerContext): Promise<TemplateResult> => {
+      return TemplateResult.redirect('www.google.com');
+    },
+  }
+}
```

Comment:
@lxndrdagreat loose idea of a template oriented version of this - it's strictly templates not API calls here hence the split. 

Going back to wanting to keep the concepts apart for now.